### DISCUSSION TITLE
Standard lead time for a part

### DISCUSSION THREAD

**Discussion by wolffam** [inventree/InvenTree#8371] [Q&A] [answered]
In my use case, it would be helpful to have a concept of a lead time so I could know how long to order and assemble an assembly depending on what parts of the build are still remaining to be purchased.

Is this a concept currently available in Inventree? I searched around and could only find a context variable in the Supplier Part docs page but I don't think that's accessible to a front end user.

Curious if others have a similar use case or know of a way to solve this problem.  Thanks all :)

**Comment by SchrodingersGat** [comment, marked as answer]
@wolffam this has been discussed but not yet implemented. It would be a good addition, although I think an early implementation would just be to add the field to the `SupplierPart` model and add some basic front-end support.

Is this something that you would be interested in helping to develop, or alternatively fund?

**Reply by wolffam** [reply]
I may have some time near the holidays to work on it. It's not an urgent feature that I need so it may be a good backlog item for me in a couple months.  Thanks!

**Comment by michelreifenrath** [comment]
Hi, we would be interested in this feature too. We are willing to participate in the development or the funding too. How is the procedure for that? Sorry, I'm new in this, so I have not an idea of the procedure.

**Reply by SchrodingersGat** [reply]
### Development

- Create a new pull request
- Read through our contribution guide
- Familiarise yourself with the existing codebase

### Funding

- Create an issue to detail the requested feature
- Add the `fund` label to the issue
- Directly fund the issue (not "paid out" until the issue is actually closed)

Either, or both, is acceptable! 

There is currently a `lead_time` attribute against the `SupplierPart` model - however there is not yet any business logic implemented to support this

**Reply by aoverlin** [reply]
@SchrodingersGat Did the Part Lead Time ever get added as an attribute to the part?   how would I amend part per "There is currently a lead_time attribute against the SupplierPart model - however there is not yet any business logic implemented to support this"?

**Reply by SchrodingersGat** [reply]
@aoverlin no, currently still no support for lead time. It needs someone to step up to specify how it might work and implement the business logic

**Reply by aoverlin** [reply]
Let me know exactly what info you need and where to send it to and I will get it over to you.  On day two of using the software and still learning capabilities,  but that lead time is essential in the system to prevent stock outs
